src/resolvers-cassandra/Edges/queries.js

```diff
@@ -76,10 +78,10 @@ function locations(args, res) { // eslint-disable-line no-unused-vars
  */
 function popularLocations(args, res) { // eslint-disable-line no-unused-vars
   return new Promise((resolve, reject) => {
-    const { period, periodType } = parseTimespan(args.timespan);
+    const { period, periodType, fromDate, toDate } = parseTimespan(args.timespan);
 
     const query = `
-    SELECT placename, placeid, mentioncount
+    SELECT placeid, mentioncount, centroidlat, centroidlon
```

Comment:
Added new feature in 4db00c6. Will also require client-side change since bbox was not considered in Fortis-v1.